src/composer/src/main.rs

```diff
@@ -47,7 +48,7 @@ pub fn exec() -> Result<(), String> {
 
     let mut sys = SystemState::new(arg1.unwrap());
     let mut build = DefaultBuilder::new();
-    build.initialize(&arg2.unwrap(), &sys)?;
+    build.initialize(&arg2.unwrap(), arg3, &sys)?;
```

Comment:
No check if `arg3` is `None` before using it. I would think that this should have caused an error. Maybe the tests aren't sufficient? I might be missing something here.

Reply:
We are using Option<String> as the third argument to pass arg3, we will check if the arg3 is none inside the build.initialize(...) function. However, we can check the arg3 at main.rs and then pass the third argument as bool as you suggested.